sql/core/src/test/resources/sql-tests/inputs/udtf/udtf.sql

```diff
@@ -16,3 +19,27 @@ SELECT * FROM udtf(1, 2) t(c1, c2), LATERAL udtf(c1, c2);
 
 -- test non-deterministic input
 SELECT * FROM udtf(cast(rand(0) AS int) + 1, 1);
+
+-- test UDTF calls that take input TABLE arguments
+SELECT * FROM UDTFCountSumLast(TABLE(t2) WITH SINGLE PARTITION);
```

Comment:
It would be really great if we could add some comments for these UDTFs (their analyze result)

Reply:
Sounds good, I ported the `AnalyzeResult` to the comments here for each UDTF.